cmake/pkg.yaml

```diff
@@ -18,19 +19,19 @@ steps:
     env:
       CC: gcc
       CXX: g++
-      LD_LIBRARY_PATH: /toolchain/lib
+      # FIXME: usrmerge in toolchain
+      LD_LIBRARY_PATH: /usr/lib64:/usr/lib:/lib64:/lib
     prepare:
       - |
         tar -xzf cmake.tar.gz --strip-components=1
 
         ./bootstrap \
-            --prefix=${TOOLCHAIN} \
+            --prefix=/usr \
             --system-libs \
             --no-system-jsoncpp \
             --no-system-libarchive \
             --parallel=$(nproc) \
-            -- -DCMAKE_BUILD_TYPE:STRING=Release \
-               -DCMAKE_INSTALL_RPATH=/toolchain/lib
+            -- -DCMAKE_BUILD_TYPE:STRING=Release || cat /tmp/build/0/Bootstrap.cmk/cmake_bootstrap.log
```

Comment:
I guess we can change `/tmp/build/0` to `./`

Reply:
will remove, that's a left in debug tool